examples/tfengine/full.hcl

```diff
@@ -12,11 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-{{$base := "../../templates/tfengine/recipes"}}
+# {{$base := "../../templates/tfengine/recipes"}}
```

Comment:
did you intend to comment out this line?

Reply:
Yeah, else I cant run terragrunt hclfmt